Rhino.Licensing/SigningAlgorithm.cs

```diff
@@ -0,0 +1,15 @@
+/// <summary>
+/// The algorithm to use when signing licenses.
+/// </summary>
+public enum SigningAlgorithm
```

Comment:
This enum is not wrapped into a namespace, which adds it to be available anytime without adding a using statement.

Should probably wrap this inside the same namespace as the `LicenseGenerator`.

Reply:
Sorted!